src/server/interfaces/model.ts

```diff
@@ -0,0 +1,8 @@
+import BaseModel from "./base";
+
+export default interface IModel extends BaseModel {
+    id?: any;
+    name: string;
+    createdAt?: Date;
```

Comment:
@Evilart86 I think createdAt & updatedAt can be moved to BaseModel

Reply:
Mb, Need to discuss